cpp/order/wigner3j.cc

```diff
@@ -9,1306 +9,2466 @@
+            0.016284840993022895, -0.05570476104834648, 0.04242787617745035, 0.007099242818288509,
+            -0.04926084132261772, 0.04926084132261772, -0.007099242818288509, -0.04242787617745035,
+            0.05570476104834648, -0.016284840993022895, -0.04426628648202024, 0.06260198269876582,
+            0.0, -0.09063800094387633, -0.09063800094387633, 0.0401710004156613,
+            0.033950691917997156, -0.06325614866282217, 0.03394087248412062, 0.020941465489420448,
+            -0.05570476104834648, 0.04541625165775246, 0.0, -0.04541625165775246,
+            0.05570476104834648, -0.020941465489420448, -0.03394087248412062, 0.06325614866282217,
+            -0.033950691917997156, -0.0401710004156613, 0.09063800094387633, 0.08124984886574907,
+            -0.06957821371079587, 0.011477428690188944, 0.04483491976341964, -0.0620459033273907,
+            0.03394087248412062, 0.016284840993022895, -0.05373723365131875, 0.05373723365131875,
+            -0.016284840993022895, -0.03394087248412062, 0.0620459033273907, -0.04483491976341964,
+            -0.011477428690188944, 0.06957821371079587, -0.08124984886574907, -0.0646604518165043,
+            0.08105616599464421, -0.054847866708622926, 0.0029582739353084648, 0.04483491976341964,
+            -0.06325614866282217, 0.04426628648202024, 0.0, -0.04426628648202024,
+            0.06325614866282217, -0.04483491976341964, -0.0029582739353084648, 0.054847866708622926,
+            -0.08105616599464421, 0.0646604518165043, 0.04461995085008539, -0.07342242818523824,
+            0.07839425928567433, -0.054847866708622926, 0.011477428690188944, 0.033950691917997156,
+            -0.06260198269876582, 0.06260198269876582, -0.033950691917997156, -0.011477428690188944,
+            0.054847866708622926, -0.07839425928567433, 0.07342242818523824, -0.04461995085008539,
+            -0.02536191581308319, 0.05176979386789814, -0.07342242818523824, 0.08105616599464421,
+            -0.06957821371079587, 0.0401710004156613, 0.0, -0.0401710004156613,
+            0.06957821371079587, -0.08105616599464421, 0.07342242818523824, -0.05176979386789814,
+            0.02536191581308319, 0.010353958773579627, -0.02536191581308319, 0.04461995085008539,
+            -0.0646604518165043, 0.08124984886574907, -0.09063800094387633, 0.09063800094387633,
+            -0.08124984886574907, 0.0646604518165043, -0.04461995085008539, 0.02536191581308319,
+            -0.010353958773579627
+        };
     }
+
     case 12:
     {
-        float wigner3j12[469]
-            = {0.00764132640913,  -0.0194816362349,  0.035645677572,    -0.0537378808656,  0.0703594019295,
-               -0.0820524576002,  0.0862629766784,   -0.0820524576002,  0.0703594019295,   -0.0537378808656,
-               0.035645677572,    -0.0194816362349,  0.00764132640913,  -0.0194816362349,  0.0420272952502,
-               -0.0633234617366,  0.0752330332119,   -0.0718102638895,  0.0517033900004,   -0.018824124
```

Comment:
"Are not"

Reply:
@vyasr I'm confused, is this comment meant to apply to this part of the file?